tensorflow/stream_executor/cuda/cuda_dnn.cc

```diff
@@ -4388,27 +4438,31 @@ class CudnnExecutionPlanRunner<void(Args...)>
 
   static port::StatusOr<CudnnExecutionPlanRunner> Create(
       GpuExecutor* parent, CudnnAccess* cudnn,
-      cudnn_frontend::ExecutionPlan plan, absl::Span<const int64_t> uids) {
+      cudnn_frontend::ExecutionPlan plan, absl::Span<const int64_t> uids,
+      bool need_side_input = true) {
```

Comment:
I am not convinced by these default params (i.e. =true here and elsewhere).  It seems like the caller Really Needs To Know whether need_side_input is true or false, otherwise Bad Things Will Happen.

Suggest removing them and making this explicit where needed.